SSA/Projects/InstCombine/AliveHandwrittenLargeExamples.lean

```diff
@@ -118,6 +119,7 @@ def alive_simplifyMulDivRem805 (w : Nat) :
   unfold MulDivRem805_lhs MulDivRem805_rhs
   simp only [simp_llvm_wrap]
   simp_alive_ssa
+  simp only [LLVM.add_reduce]
```

Comment:
We should not need to add this simp only rewrite explicitly here, instead it should be executed as either `simp_alive_ssa` or `simp_alive_undef`.